lib/project/cordova-project.ts

```diff
@@ -168,10 +169,13 @@ export class CordovaProject extends FrameworkProjectBase implements Project.IFra
 
 	private ensureCordovaJs(platform: string, projectDir: string, frameworkVersion: string): IFuture<void> {
 		return (() => {
-			let cordovaJsFileName = path.join(projectDir, util.format("cordova.%s.js", platform).toLowerCase());
-			if (!this.$fs.exists(cordovaJsFileName).wait()) {
+			let cordovaJsFileName = path.join(projectDir, `cordova.${platform.toLowerCase()}.js`),
```

Comment:
Can you change the name of `cordovaJsFileName ` to `cordovaJsFilePath` because this variable contains the path to the file not the name of it.